Nikita_Korzhavin/api/calculateSmthInArray.js

```diff
@@ -0,0 +1,13 @@
+function calculateSmth(param, array) {
+  if (param === undefined || array === undefined) {
+    throw new Error('smth wrong in calculateSmth function');
```

Comment:
As well as this error, you have a particular condition for throwing this error, why just won't describe it?

Reply:
fixed